app/Policies/ProjectPolicy.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace REBELinBLUE\Deployer\Policies;
+
+use REBELinBLUE\Deployer\User;
+use REBELinBLUE\Deployer\Project;
+use Illuminate\Auth\Access\HandlesAuthorization;
+
+class ProjectPolicy
+{
+    use HandlesAuthorization;
+
+    public function before($user, $ability)
+    {
+        if ($user->is_admin === 1) {
```

Comment:
Add `is_admin` to the `casts` array on the `User` entity to cast it to a boolean then you can just do `$user->is_admin`

Reply:
Done

Reply:
Oops, breaks my code.

Reply:
fixed